src/lightning_classify/models/model_builder.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any
+
+import torch
+from torch import nn
+from torchvision import models
+from torch.optim import Adam
+import lightning as L
+from torchmetrics import Accuracy
+
+
+class ImageRecogModel(L.LightningModule):
+    def __init__(self, lrate: float, num_classes: int = 18) -> Any:
+        super().__init__()
+        self.num_classes = num_classes
+        self.lrate = lrate
+
+        self.model = self._load_backbone()
+        self.criterion = nn.CrossEntropyLoss()
+        self.metrics = Accuracy(task="multiclass", num_classes=self.num_classes)
```

Comment:
you need to have more than one metrics, remove the self.metrics with this example code

```python
self.train_acc = Accuracy(task="multiclass", num_classes=self.num_classes)
self.val_acc = Accuracy(task="multiclass", num_classes=self.num_classes)
```